Makefile

```diff
@@ -147,6 +147,9 @@ fix-install-permissions:
 fix-xcode-select:
 	sudo xcode-select -s $(XCODE_DEVELOPER_ROOT)
 
+fix-xcode-first-run:
+	$(XCODE_DEVELOPER_ROOT)/usr/bin/xcodebuild -runFirstLaunch
+
```

Comment:
> Why copy paste a command?

because no one will know about that make target ? ;-)

Reply:
I can add a little string :)